### DISCUSSION TITLE
mi calander

### DISCUSSION THREAD

**Discussion by Gaganshettyy** [reiryuki/Miui-Core-Magisk-Module#14] [Q&A] [answered]
i have been using miui stock i have recently switched to custom to but the calander app in miui was perfect can do something about it the remainder and birthday events are all synced in the mi cloud iw there any way to to install mi calander

**Comment by reiryuki** [comment, marked as answer]
You can normal install mi calender from play store with the latest miui core version.

Grant permissions manually.
